media/starboard/BUILD.gn

```diff
@@ -54,7 +55,30 @@ source_set("starboard") {
     "COBALT_MEDIA_ENABLE_UMA_METRICS=0",
   ]
 
-  sources = []
+  if (media_use_ffmpeg) {
```

Comment:
can remove this conditional and directly start with if(!media_use_ffmpeg)

Reply:
Done.